src/pages/writing-nft/index.vue

```diff
@@ -34,28 +34,34 @@
           v-t="'campaign_nft_page_description'"
           class="mt-[16px] max-w-[536px] mx-auto"
         />
-        <svg
-          class="mt-[24px] mx-auto text-like-green"
-          width="40"
-          height="86"
-          viewBox="0 0 40 86"
-          fill="none"
-          xmlns="http://www.w3.org/2000/svg"
-        >
-          <path
-            fill-rule="evenodd"
-            clip-rule="evenodd"
-            d="M21 82.0858L38.2929 64.7929C38.6834 64.4024 39.3166 64.4024 39.7071 64.7929C40.0976 65.1834 40.0976 65.8166 39.7071 66.2071L20.7071 85.2071C20.3166 85.5976 19.6834 85.5976 19.2929 85.2071L0.292893 66.2071C-0.0976312 65.8166 -0.0976308 65.1834 0.292893 64.7929C0.683417 64.4024 1.31658 64.4024 1.70711 64.7929L19 82.0858L19 1C19 0.447715 19.4477 0 20 0C20.5523 0 21 0.447715 21 1L21 82.0858Z"
-            fill="currentColor"
-          />
-        </svg>
       </div>
-      <a
-        class="flex items-center justify-center mt-[24px] text-medium-gray hover:text-like-cyan-dark text-[12px] leading-[5/3] underline transition-colors cursor-pointer"
-        href="https://likecoin.github.io/likecoin-nft-dashboard/"
-        target="_blank"
-        rel="noopener"
-      >{{ $t('campaign_nft_view_nft_dashboard') }}</a>
+
+      <nav class="mt-[24px] relative flex-col laptop:flex-row flex items-center justify-center self-stretch gap-[32px]">
```

Comment:
I think the navbar is too close to the intro section
```suggestion
      <nav class="mt-[48px] relative flex-col laptop:flex-row flex items-center justify-center self-stretch gap-[32px]">
```